sound/soc/intel/boards/Kconfig

```diff
@@ -536,6 +536,30 @@ config SND_SOC_INTEL_TGL_RT711_RT1308_MACH
 
 endif ## SND_SOC_SOF_TIGERLAKE && SND_SOC_SOF_INTEL_SOUNDWIRE
```

Comment:
Commit message: ASoC: Intel: boards: support Elkhart Lake with rt5660

Reply:
I will update it.